config/sql/se/Artists.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `Artists` (
+  `ArtistId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `DeathYear` datetime DEFAULT NULL,
```

Comment:
Is there much value in storing this as a `datetime` rather than `int`? We're only asking for the year, so the month, day & time portions will all be unused.